angularjs/angular.d.ts

```diff
@@ -1761,19 +1757,29 @@ declare module angular {
         compile?: IDirectiveCompileFn;
         controller?: any;
         controllerAs?: string;
-        bindToController?: boolean|Object;
+        /**
+         * @deprecated
+         * Deprecation warning: although bindings for non-ES6 class controllers are currently bound to this before
+         * the controller constructor is called, this use is now deprecated. Please place initialization code that
+         * relies upon bindings inside a $onInit method on the controller, instead.
+         */
+        bindToController?: boolean | Object;
```

Comment:
I stumbled upon this, as I am currently diving into directives. From my understanding, the `@deprecated` annotation is wrong. The behaviour for controllers implemented as non-ES6 classes might change, but this attribute itself is not deprecated. (https://docs.angularjs.org/api/ng/service/$compile#-bindtocontroller-)

Truth do be told, this text is a good hint, worth to know. But I wouldn't either know how to annotate it..